includes/class-amp-theme-support.php

```diff
@@ -411,23 +411,23 @@ public static function ensure_proper_amp_location() {
 			global $wp;
 			$path_args = [];
 			wp_parse_str( $wp->matched_query, $path_args );
-			if ( isset( $path_args[ amp_get_slug() ] ) && '' !== $path_args[ amp_get_slug() ] ) {
+			if ( isset( $path_args[ amp_get_slug() ] ) && '1' !== $path_args[ amp_get_slug() ] ) {
```

Comment:
So this change should be reverted.

```suggestion
			if ( isset( $path_args[ amp_get_slug() ] ) && '' !== $path_args[ amp_get_slug() ] ) {
```